mclazy.py

```diff
@@ -236,9 +240,31 @@ def main():
             installed_pkgs[h['name']] = h['version']
         print_debug(f"Loaded rpmdb with {len(installed_pkgs)} items")
 
-    # parse the configuration file
+    # parse the branches.xml file
+    branches = BranchesXml(args.branches)
+
+    if args.fedora_branch not in branches:
```

Comment:
Something we're missing: `args.fedora_branch` might be an alias (i.e. "newstable"). But then the rest of mclazy uses `args.fedora_branch`, which of course won't work right given an alias

So we should include a remapping of alias -> real branch name here. `args.fedora_branch = branches[args.fedora_branch].name`